app/common/state/ledgerState.js

```diff
@@ -65,12 +68,28 @@ const ledgerState = {
     return state.setIn(['ledger', 'locations', url, prop], value)
   },
 
+  getVerifiedPublisherLocationProp: (state, url) => {
```

Comment:
`getVerifiedPublisherLocationProp` -> `getVerifiedPublisherLocation`

Let's rename it, because we don't have prop anymore

Reply:
Done